ios/Barometer.m

```diff
@@ -56,30 +57,47 @@ - (void) isAvailableWithResolver:(RCTPromiseResolveBlock) resolve
     NSLog(@"Can not set update interval for barometer, doing nothing");
 }
 
+RCT_EXPORT_METHOD(setLogLevel:(int) level) {
+    if (level > 0) {
+        NSLog(@"setLogLevel: %f", level);
+    }
+
+    self->logLevel = level;
+}
+
 RCT_EXPORT_METHOD(getUpdateInterval:(RCTResponseSenderBlock) cb) {
     NSLog(@"getUpdateInterval is not meaningul for a barometer sensor");
     cb(@[[NSNull null], [NSNumber numberWithDouble:0.0]]);
 }
 
 RCT_EXPORT_METHOD(startUpdates) {
-    NSLog(@"startUpdates");
+    if (self->logLevel > 0) {
+        NSLog(@"startUpdates/startRelativeAltitudeUpdates");
+    }
 
     [self->_altimeter startRelativeAltitudeUpdatesToQueue:[NSOperationQueue mainQueue] withHandler:^(CMAltitudeData * _Nullable altitudeData, NSError * _Nullable error) {
         if (error) {
             NSLog(@"error while getting sensor data");
         }
-        
+
         if (altitudeData) {
+            if (self->logLevel > 1) {
+                NSLog(@"Updated altitue value: %f, %f", altitudeData.pressure.doubleValue, timestamp);
```

Comment:
Timestamp is missing here, since this PR our app is not building anymore. Undeclared identifier timestamp.